sdk/sdkclients/sdkdeploymentsclient/client.go

```diff
@@ -69,41 +70,44 @@ func (c *client) Test(ctx context.Context, id sdktypes.DeploymentID) error {
 	return nil
 }
 
-// Create implements sdkservices.Deployments.
-func (c *client) Create(ctx context.Context, deployment sdktypes.Deployment) (sdktypes.DeploymentID, error) {
-	resp, err := c.client.Create(ctx, connect.NewRequest(&deploymentsv1.CreateRequest{Deployment: deployment.ToProto()}))
+// Drain implements sdkservices.Deployments.
+func (c *client) Drain(ctx context.Context, id sdktypes.DeploymentID) error {
+	resp, err := c.client.Drain(ctx, connect.NewRequest(&deploymentsv1.DrainRequest{DeploymentId: id.String()}))
 	if err != nil {
-		return nil, rpcerrors.TranslateError(err)
+		return rpcerrors.TranslateError(err)
```

Comment:
can probably add a FromConnectError matching the ToConnectError to have it more zen
WDYT ?

Reply:
sure, worth a ticker. This is the first step into this direction indeed. But we are still not there. Need to fix tests, then write tests for all commands and then ensure the right errors. 
I think I will finish with all commands first